openshift/generate-manifests.sh

```diff
@@ -20,6 +20,22 @@ IMAGE_MAPPINGS[kube-rbac-proxy]='${KUBE_RBAC_PROXY_IMAGE}'
 # shellcheck disable=SC2016
 IMAGE_MAPPINGS[manager]='${OPERATOR_CONTROLLER_IMAGE}'
 
+
+##################################################
+# Determine if using gsed (for macOS) or sed
+##################################################
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  if command -v gsed >/dev/null 2>&1; then
+    SED_CMD="gsed"
+  else
+    echo "gsed is not installed. Please install it using 'brew install gnu-sed'."
+    exit 1
+  fi
+else
+  SED_CMD="sed"
+fi
+
```

Comment:
On macOS, this should not be necessary, you just need to add a blank argument after the `-i` option.